cogs/events/events.py

```diff
@@ -493,6 +506,8 @@ def __init__(self, bot):
 def setup(bot):
     async def message_events(message):
         clean_message = message.clean_content.lower()
+        # MM: Added so list instead of string
+        mset = clean_message.split(' ')
```

Comment:
change this name from `mset` to `message_split` - something that's actually useful